packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -109,7 +118,26 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
             />
           ))}
         <div className={styles.text}>
-          <Linkify options={{ target: '_blank' }}>{message.message}</Linkify>
+          <Linkify
+            options={{
+              attributes: {
+                onClick: (event: React.MouseEvent<HTMLAnchorElement>) => {
+                  const url = event.currentTarget.href
+
+                  if (isAudiusUrl(url)) {
+                    const path = getPathFromAudiusUrl(url)
+                    event.nativeEvent.preventDefault()
+                    onClickInternalLink(path ?? '/')
+                  }
```

Comment:
nah there wasn't an external click function here to begin with. might need to be added though, but didn't want to touch this in case that was intentional